debezium-core/src/test/java/io/debezium/jdbc/JdbcConnectionTest.java

```diff
@@ -28,29 +28,34 @@
 import org.junit.Test;
 
 import io.debezium.jdbc.JdbcConnection.ConnectionFactory;
+import io.debezium.util.ThreadNameContext;
 
 public class JdbcConnectionTest {
 
     @Test
     public void testNormalClose() throws SQLException {
         ConnectionFactory connFactory = (config) -> new NormalConnection();
-        JdbcConnection conn = new JdbcConnection(JdbcConfiguration.empty(), connFactory, "\"", "\"");
+
+        JdbcConnection conn = new JdbcConnection(JdbcConfiguration.empty(), connFactory, "\"", "\"",
+                new ThreadNameContext("test-connector", "${debezium}-${connector.name}-${task.id}", "0"));
```

Comment:
Let's add ${connector.name} and ${task.id} on top of the default thread name pattern in this class too.